### PR TITLE
[fix]: my incidents login page acc feedback

### PR DESCRIPTION
**Pull request by vdegraaf**
Ticket: [SIG-4757](https://datapunt.atlassian.net/browse/SIG-4757)

This PR implements the feedback from the acceptance environment version 2.10.1.

## Signalen

Before opening a pull request, please ensure:

- [x] Double-check your branch is based on `main` and targets `main`
- [x] Pull request has tests (we are going for 100% coverage!)
- [x] Code is well-commented, linted and follows project conventions
- [x] Committed source code is headed by the correct SPDX license expression

Be kind to code reviewers, please try to keep pull requests as small and focused as possible :)

### CODE REVIEW

File: src/signals/my-incidents/pages/BasePage.tsx [hunk 1]
```diff
@@ -22,24 +29,34 @@ export const BasePage: FC<Props> = ({
...
+      </Helmet>
+
+      <article>
+        {pageTitle && (
+          <header>
+            <StyledHeading>{pageTitle}</StyledHeading>
+          </header>
+        )}
+        {paragraphs &&
+          paragraphs.map((paragraph, index) => (
```
Comment (OscarBakker) [inline src/signals/my-incidents/pages/BasePage.tsx:50]: kan dit niet gewoon paragraphs?. zijn, het wordt dan null

File: src/signals/my-incidents/pages/BasePage.tsx [hunk 2]
```diff
@@ -22,24 +29,34 @@ export const BasePage: FC<Props> = ({
...
+            <Paragraph key={index} fontSize={16}>
```
Comment (OscarBakker) [inline src/signals/my-incidents/pages/BasePage.tsx:51]: would be much better to make this 1 rem, and set the base font size to 16. there is is 27 matches for '16px'. 

https://datapunt.atlassian.net/browse/SIG-4889

tbc!

**Review by OscarBakker — APPROVED** [review]
Ziet er goed uit, 1 ding

**Review by SireeKoolenWijkstra — COMMENTED** [review]
image
Moest de afstand niet 16px zijn en is ie hier 24px?

File: src/signals/my-incidents/pages/BasePage.tsx (hunk 1 above)
Comment (vdegraaf) [reply src/signals/my-incidents/pages/BasePage.tsx:50]: Ik vind het zo fijner lezen bij view components, maar heb het veranderd

File: src/signals/my-incidents/pages/BasePage.tsx (hunk 2 above)
Comment (vdegraaf) [reply src/signals/my-incidents/pages/BasePage.tsx:51]: Ja goeie!

**Pull request merged**